libs/core/breadcrumb/breadcrumb.component.ts

```diff
@@ -147,10 +147,8 @@ export class BreadcrumbComponent implements AfterViewInit, HasElementRef {
      * if original element had router link we are proxying click to that element
      * */
     itemClicked(breadcrumbItem: BreadcrumbItemComponent, $event: Event): void {
-        if (breadcrumbItem._needsClickProxy) {
```

Comment:
if conditional check is no longer needed, please update function doc descriptions too.